tests/test_app.py

```diff
@@ -190,11 +180,10 @@ def test_it_works_with_api_p(self) -> None:
     # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
     def test_it_works_n(self) -> None:
-        proc = subprocess.Popen([sys.executable, "-m", "credsweeper"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        _stdout, stderr = proc.communicate()
+        _stdout, _stderr = self._m_credsweeper([])
 
-        # Merge more than two whitespaces into one because stdout and stderr are changed based on the terminal size
-        output = " ".join(stderr.decode("UTF-8").split())
+        # Merge more than two whitespaces into one because _stdout and stderr are changed based on the terminal size
```

Comment:
```suggestion
        # Merge more than two whitespaces into one because _stdout and _stderr are changed based on the terminal size
```
To be consistent